src/components/content/content.stories.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, select, boolean } from '@storybook/addon-knobs';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import notes from './notes.md';
+import Content from './content.js';
+
+storiesOf('Content', module)
+  .add('default', () => {
+    const children = text('children', 'An example of some content.');
+    const title = text('title', 'Content Component');
+    const knobAs = select('as', OptionsHelper.themesBinary, Content.defaultProps.as);
+    const inline = boolean('inline', Content.defaultProps.inline);
+    const align = select('align', OptionsHelper.alignFull, Content.defaultProps.align);
+    const titleWidth = inline ? text('titleWidth', '') : undefined;
+    const bodyFullWidth = boolean('bodyFullWidth', Content.defaultProps.bodyFullWidth);
+
+    return (
+      <Content
```

Comment:
We shouldn't see `@class @constructor`.

![image](https://user-images.githubusercontent.com/26925206/53880259-7c401000-4008-11e9-85ef-f4fa768a8704.png)

Reply:
thanks, done